### PR TITLE
feat(setting roles): setting up roles

### PR DESCRIPTION
**Pull request by James-nyagatare**
## What does this PR do?
- Assigns roles to any staff member

## Description of Task to be completed?
```
- Assigns roles to any staff member
- Documenting the route
- Write unit testing of the endpoint
```

## How should this be manually tested?
- Using postman

## What are the relevant pivotal tracker stories?
[Setting roles](https://www.pivotaltracker.com/story/show/176153388)

## Questions:
N/A

### CODE REVIEW

File: src/controllers/admin.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import models from '../database/models';
+import managers from '../helpers/manager';
+import roles from '../helpers/roles';
+
+export default class AdminController {
+  static async updateRole(req, res) {
+    try {
+      const { newRole } = req.body;
+      const { email } = req.params;
+      const user = await models.User.findOne({ where: { email } });
```
Comment (harerakalex) [inline src/controllers/admin.js]: As we agreed, better do not use models in controllers, instead, create a separate file in a folder called queries or services and call it in the controller.

**Review by harerakalex — CHANGES_REQUESTED** [review]
Great start @James-nyagatare

File: src/controllers/admin.js (hunk 1 above)
Comment (James-nyagatare) [reply src/controllers/admin.js]: @harerakalex it have been corrected. you can check and let me know if I missed something.

File: src/routes/welcomeRoute.js [hunk 2]
```diff
@@ -21,9 +20,4 @@ const router = Router();
 
 router.get('/', (_req, res) => Response.success(res, 200, 'Welcome to barefoot nomand'));
```
Comment (Mucyo-Aime-Christian) [inline src/routes/welcomeRoute.js]: better use status code names

**Review by Mucyo-Aime-Christian — COMMENTED** [review]
LGTM but try to check the comment that i left on your file

File: src/helpers/statusCode.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+const code = {
+  ok: 200,
```
Comment (harerakalex) [inline src/helpers/statusCode.js:2]: Nice idea @James-nyagatare

**Review by harerakalex — APPROVED** [review]
LGTM @James-nyagatare

File: src/routes/welcomeRoute.js (hunk 2 above)
Comment (James-nyagatare) [reply src/routes/welcomeRoute.js]: Yeah, It's resolved @Mucyo-Aime-Christian

**Pull request merged**